tests/ui/specs/VirtualList/VirtualGridList/VirtualGridListPage.js

```diff
@@ -133,10 +135,45 @@ class VirtualGridListPage extends Page {
 		}, scrollableSelector);
 	}
 
-	itemOffsetTopById (id) {
-		return browser.execute(function (_element) {
-			return _element.getBoundingClientRect().top;
-		}, this.item(id).value);
+	itemOffsetTopById (index) {
+		return browser.execute(function (_listItemSelector, _scrollableSelector, _index) {
+			const itemContent = document.querySelectorAll(_listItemSelector)[_index];
+			const scroller = document.querySelector(_scrollableSelector);
+			return Math.round(scroller.getBoundingClientRect().height - itemContent.getBoundingClientRect().top);
+		}, listItemSelector, scrollableSelector, index);
+	}
+
+	getItemSize (index = 0) {
+		return browser.execute(function (_listItemSelector, _index) {
+			const itemContent = document.querySelectorAll(_listItemSelector)[_index];
+			const itemHeight = itemContent.getBoundingClientRect().height;
+			const itemWidth = itemContent.getBoundingClientRect().width;
```

Comment:
How about using destructing to avoid duplicate function calls
```suggestion
			const {height: itemHeight, width: itemWidth}  = itemContent.getBoundingClientRect();
```

Reply:
Suggested change.